### PR TITLE
updated mitmproxy (0.17.1)

### PR DESCRIPTION
**Pull request by rogeriopradoj**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/mitmproxy.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'mitmproxy' do
-  version '0.15'
-  sha256 'b071e758be27005f10e580e3ca7b194d475fe8e57b971f36b44275234a7617d6'
+  version '0.17.1'
+  sha256 'dd5b2779f078c3208c02c673ee6343bf7aedf1c1759df20f8103241147d833ea'
 
-  url "https://mitmproxy.org/download/osx-mitmproxy-#{version}.tgz"
+  url "https://github.com/mitmproxy/mitmproxy/releases/download/v#{version}/mitmproxy-#{version}-osx.tar.gz"
```
Comment (adidalal) [inline Casks/mitmproxy.rb]: Needs a `# github.com/mitmproxy/mitmproxy was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (adidalal) [inline Casks/mitmproxy.rb]: Also, mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 09fd670b8a9a046e6a6f038b8ef2b21dce9a6edb. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by rogeriopradoj** [comment]
Thanks, @adidalal !

**Pull request closed without merging**